script/bundle-libraries.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -xe
+
+yarn purs-backend-es bundle-module -m App.LoadFigure --to dist/fluid/load-figure.js
```

Comment:
For consistency maybe we should extract `./util/bundle-module.sh` (analogous to `./util/bundle.sh`) and call that here instead (with the extra `--platform=node` argument in the second case).